src/Core/Geographic/Crs.js

```diff
@@ -2,19 +2,29 @@ import proj4 from 'proj4';
 
 proj4.defs('EPSG:4978', '+proj=geocent +datum=WGS84 +units=m +no_defs');
 
+function isString(s) {
+    return typeof s === 'string' || s instanceof String;
+}
+
 function isTms(crs) {
-    return crs.startsWith('TMS');
+    return isString(crs) && crs.startsWith('TMS');
 }
 
 function isEpsg(crs) {
-    return crs.startsWith('EPSG');
+    return isString(crs) && crs.startsWith('EPSG');
 }
 
 function formatToTms(crs) {
+    if (!isString(crs)) {
+        throw new Error(`Crs parameter value must be a string: '${crs}'`);
+    }
```

Comment:
may be encapsulate in method `mustBeString`?

Reply:
yes, done